Tests/LinuxMain.swift

```diff
@@ -1,7 +1,6 @@
-// Generated using Sourcery 1.0.0 — https://github.com/krzysztofzablocki/Sourcery
+// Generated using Sourcery 1.3.4 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
the sourcery task is going to fail on evg because of this; once my PR #614 is merged in you can rebase on top of it and that should fix it.